packages/sitecore-jss-cli/src/create/index.ts

```diff
@@ -65,12 +65,14 @@ export function applyHostNameToSitecoreConfig(configXml: string, hostName: strin
  * @param {string} name App name
  * @param {string} hostName App hostname
  * @param {string} replaceName Name value to replace
+ * @param {string} withPrefix Option to pass to replacePrefix - defaults to false
  */
 export function applyNameToProject(
   projectFolder: string,
   name: string,
   hostName: string,
-  replaceName: string
+  replaceName: string,
+  withPrefix?: string
```

Comment:
Should this be a boolean?

Reply:
I had it as a boolean originally but it was still coming in as a string?